<launch>
	<node name="centralhub" pkg="centralhub" type="centralhub" output="screen" />

	<param name="connect" value="true" type="bool" />
	<param name="db_url" value="https://my-project-1516369881504.firebaseio.com/sessions/15.json"/>
	<param name="persist" value="true" type="bool" />

	<param name="path" value="centralhub_output.csv" />

</launch>